### PR TITLE
API Remove deprecated code

### PR DESCRIPTION
**Pull request by sabina-talipova**
## Parent issue
- https://github.com/silverstripe/silverstripe-framework/issues/10546

### CODE REVIEW

File: src/BaseHandlerTrait.php [hunk 1]
```diff
@@ -21,54 +18,31 @@
...
-    /**
-     * @param Decoder $decoder
      * @return AttestationStatementSupportManager
      */
-    protected function getAttestationStatementSupportManager(Decoder $decoder): AttestationStatementSupportManager
+    protected function getAttestationStatementSupportManager(): AttestationStatementSupportManager
     {
         $manager = new AttestationStatementSupportManager();
         $manager->add(new NoneAttestationStatementSupport());
-        $manager->add(new FidoU2FAttestationStatementSupport($decoder));
```
Comment (emteknetnz) [inline src/BaseHandlerTrait.php:48]: I'm unsure if we're supposed to be removing this here

**Review by emteknetnz — COMMENTED** [review]
This one's probably blocked on https://github.com/silverstripe/silverstripe-webauthn-authenticator/issues/124 since I don't think we can manually test this yet

File: src/BaseHandlerTrait.php (hunk 1 above)
Comment (sabina-talipova) [reply src/BaseHandlerTrait.php:48]: DONE

**Review by emteknetnz — CHANGES_REQUESTED** [review]
Could you please create a new issue for this and put this PR on this, we're not going to be able to merge this any time soon given the poor state the CMS 5 version of this module is in right now i.e. we cannot test it

**Comment by sabina-talipova** [comment]
Created new issue for this PR
- https://github.com/silverstripe/silverstripe-webauthn-authenticator/issues/127

**Review by emteknetnz — APPROVED** [review]
Tested locally, it works

I had both of these PRs present as well:
- https://github.com/silverstripe/silverstripe-mfa/pull/484
- https://github.com/silverstripe/silverstripe-webauthn-authenticator/pull/128

**Pull request merged**